data/raw/hanhu_data/f_4433_hanhu.py

```diff
@@ -0,0 +1,131 @@
+import ctypes
+import hashlib
+import binascii
+
+def f_4435(filepath):
+    """
+    Loads a DLL file from a given filepath, calculates its MD5 and SHA256 hashes,
+    and prints these hashes in hexadecimal format. This function is a demonstration
+    of file handling, usage of the hashlib library for hash calculations, and binascii
+    for hexadecimal conversion. Note that the actual operations performed on the loaded
+    DLL are limited to hash calculation.
+
+    Parameters:
+    filepath (str): The path of the DLL file.
+
+    Returns:
+    str: The name of the loaded DLL file.
+
+    Requirements:
+    - ctypes
+    - hashlib
+    - binascii
+
+    Examples:
+    Examples:
+    >>> with open('libc.so.6', 'w') as f:
+    ...     _ = f.write("")
+    >>> result = f_4435('libc.so.6')
+    MD5 Hash: d41d8cd98f00b204e9800998ecf8427e
+    SHA256 Hash: e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
+    >>> isinstance(result, str) 
+    True
+    >>> 'libc.so.6' in result
+    True
+    >>> import os
```

Comment:
`remove` won't be necessary for these interactive examples, as we won't execute them when evaluating models

Reply:
When checking using pytest --doctest-modules, it will write a file, right? 

Is this behavior okay?

Reply:
As `pytest --doctest-modules` is done in the container in GitHub Actions in the end, no additional files will be added to the data folder. However, if you are testing locally, please make sure that you are not adding any files or folders created due to the doctest.